test/input_gen/genModelsRecurrent_v2.py

```diff
@@ -172,6 +172,34 @@ def forward(self, inputs, labels):
         loss = self.loss(ret, labels[0])
         return ret, loss
 
+class GRUCellFC(torch.nn.Module):
+    def __init__(self, unroll_for=2, num_grucell=1):
+        super().__init__()
+        self.input_size = self.hidden_size = 2
+        self.grus = torch.nn.ModuleList(
+            [
+                torch.nn.GRUCell(self.input_size, self.hidden_size, bias=True)
+            ]
+        )
+        self.fc = torch.nn.Linear(2,2)
+        self.unroll_for = unroll_for
+        self.loss = torch.nn.MSELoss()
+
+    def forward(self, inputs, labels):
+        out = inputs[0]
+        hs = inputs[1:]
+        ret = []
+        for _ in range(self.unroll_for):
+            for i, (gru, h) in enumerate(zip(self.grus, hs)):
+                hs[i] = gru(out, h)
+                out = hs[i]
```

Comment:
I think only final hidden value is used for fc layer.
`out = hs[self.hidden_size]`  after iteration will be enough.

Reply:
yes. I changed the code further. Thanks.